src/telliot_feeds/reporters/customized/backup_reporter.py

```diff
@@ -0,0 +1,184 @@
+        if updated_at == 0 or int(updated_at) > current_timestamp:
+            logger.info(f"chainlink is broken. updated at: {updated_at}")
+            return True
+        if answer == 0:
+            logger.info(f"chainlink is broken. answer: {answer}")
+            return True
+        return False
+
+    async def get_tellor_latest_data(self) -> Optional[GetDataBefore]:
+        if self.datafeed is None:
+            logger.debug(f"no datafeed set: {self.datafeed}")
+            return None
+        data, status = await self.oracle.read("getDataBefore", self.datafeed.query.query_id, self.current_time())
+        if not status.ok:
+            logger.warning(f"error getting tellor data: {status.e}")
+            return None
+        return GetDataBefore(*data)
+
+    async def conditions_met(self) -> bool:
+        """Trigger methods to check conditions if reporting spot is necessary"""
+        logger.info("checking conditions and reporting if necessary")
+        tellor_data = await self.get_tellor_latest_data()
+
+        current_time = self.current_time()
+        time_passed_since_tellor_report = current_time - tellor_data.timestampRetrieved if tellor_data else current_time
+        if (
+            tellor_data is None  # if there is tellor data check chainlink feeds
+            or tellor_data.retrieved is False
+            # if tellor data is old check chainlink feeds
+            or time_passed_since_tellor_report > self.chainlink_is_frozen_timeout
+        ):
+            chainlink_latest_round_data = self.get_chainlink_latest_round_data()
+
+            if chainlink_latest_round_data is None:
+                return True
+            if self.chainlink_is_broken(chainlink_latest_round_data):
+                return True
+            if self.chainlink_is_frozen(chainlink_latest_round_data):
+                return True
+            chainlink_previous_round_data = self.get_chainlink_previous_round_data(chainlink_latest_round_data.roundId)
+
+            if chainlink_previous_round_data is None:
+                return True
+
+            if self.chainlink_price_change_above_max(chainlink_latest_round_data, chainlink_previous_round_data):
+                return True
+            logger.info(f"chainLink {self.datafeed.query.descriptor} data is recent enough")  # type: ignore
+            return False
+        else:
+            logger.info(f"tellor {self.datafeed.query.descriptor} data is recent enough")  # type: ignore
+            return False
+
+    async def report(self, report_count: Optional[int] = None) -> None:
+        """Submit values to Tellor oracles on an interval."""
+
+        while report_count is None or report_count > 0:
+            online = await self.is_online()
+            if online:
+                if self.has_native_token() and await self.conditions_met():
+                    _, _ = await self.report_once()
+            else:
+                logger.warning("Unable to connect to the internet!")
```

Comment:
Update logging to notify that it's not reporting due to not `conditions_met()` vs no internet condition.